tests/VolleyManagement.UnitTests/Mvc/Controllers/GameResultsControllerTests.cs

```diff
@@ -91,9 +92,12 @@ public void CreatePostAction_ValidModel_Created()
 
             // Act
             var result = sut.Create(gameResult) as RedirectToRouteResult;
+            string expectedDisplayGameNumber = "G" + gameResult.GameNumber;
 
             // Assert
             TestHelper.AreEqual(expectedGameResult, actualGameResult, new GameComparer());
+            Assert.AreEqual(expectedDisplayGameNumber, gameResult.DisplayGameNumber);
```

Comment:
Why existing code did not suit you?

Reply:
What do you mean? I add new 3 line to check if the DisplayGameNumber return correct data and to check if DisplayGameNumbe not equal GameNumber.

Reply:
Existing code don't check if property DisplayGameNumber return correct data, so that's why I add 3 new line to check it.  Of courese I can add in GameComparer to check if the DispalyGameNumber of one games equal to another, but when if there are mistake of logic of displaying number of game, the test don't fail.